src/InjectionException.php

```diff
@@ -8,7 +8,7 @@ final class InjectionException extends InjectorException
      * Add a human readable version of the invalid callable to the standard 'invalid invokable' message.
      *
      * @param array           $inProgressMakes
-     * @param                 $callableOrMethodStr
+     * @param callable        $callableOrMethodStr
```

Comment:
Callable isn't really correct here, because it's really not callable. Maybe mixed?

Reply:
What could be in `$callableOrMethodStr`?

Reply:
All these are callable: https://phpstan.org/writing-php-code/phpdoc-types#callables

Reply:
I see! As per method name the received value is an invalid callable, which **may** be a callable but not sure!

Reply:
There is no 'else' only 'elseif'.

Reply:
Done.